[{"ContainerToolsException.Message":"Visual Studio Container Tools requires Docker Desktop.  To get it, see https://go.microsoft.com/fwlink/?linkid=847268  For more info, please see: https://aka.ms/DockerToolsTroubleshooting","ContainerToolsException.ErrorCode":"","ContainerToolsException.StackTrace":"","ContainerToolsException.HResult":"-2146233088","ContainerToolsException.InnerException":"Microsoft.VisualStudio.Containers.Tools.Common.ContainerToolsException: Unable to run 'docker-compose'. Verify that Docker Desktop is installed and running locally. For troubleshooting, please refer to https://aka.ms/DockerToolsTroubleshooting. ---> System.ComponentModel.Win32Exception: O sistema não pode encontrar o arquivo especificado\r\n   em System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)\r\n   em System.Diagnostics.Process.Start()\r\n   em Microsoft.Docker.Utilities.CommandLineClient.<>c__DisplayClass0_0.<ExecuteAsync>b__0() na D:\\a\\_work\\1\\s\\src\\Microsoft.Docker\\Utilities\\CommandLineClient.cs:linha 172\r\n   --- Fim do rastreamento de pilha de exceções internas ---\r\n   em System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   em Microsoft.Docker.DockerComposeClient.<ExecuteAsync>d__13.MoveNext() na D:\\a\\_work\\1\\s\\src\\Microsoft.Docker\\DockerComposeClient.cs:linha 253\r\n--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---\r\n   em System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   em Microsoft.Docker.DockerComposeClient.<DownAsync>d__8.MoveNext() na D:\\a\\_work\\1\\s\\src\\Microsoft.Docker\\DockerComposeClient.cs:linha 104\r\n--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---\r\n   em System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   em Microsoft.Docker.DockerWorkspace.<ComposeDownContainersAsync>d__32.MoveNext() na D:\\a\\_work\\1\\s\\src\\Microsoft.Docker\\DockerWorkspace.cs:linha 424\r\n--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---\r\n   em System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   em Microsoft.Docker.BuildTasks.DockerBuildTask.<ExecuteAsync>d__34.MoveNext() na D:\\a\\_work\\1\\s\\src\\Microsoft.Docker.BuildTasks\\DockerBuildTask.cs:linha 82\r\nCódigo de erro: DT1001\r\nGravidade: Error\r\nLinha de comando: docker-compose  -f \"C:\\Desenvolvimento\\C#\\ApiProLife\\docker-compose.yml\" -f \"C:\\Desenvolvimento\\C#\\ApiProLife\\docker-compose.override.yml\" -f \"C:\\Desenvolvimento\\C#\\ApiProLife\\obj\\Docker\\docker-compose.vs.release.partial.g.yml\" -p dockercompose15640030004284940075 --ansi never kill\r\n"}]